src/SmartTransactionsController.test.ts

```diff
@@ -17,8 +17,8 @@ import {
 import nock from 'nock';
 import * as sinon from 'sinon';
 
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
It's unclear why these were added, but they are no longer needed

Reply:
Oh I see, `yarn tsc` passes but `yarn build` fails due to the tsconfig root directory. I will fix.

Reply:
https://github.com/MetaMask/smart-transactions-controller/pull/468/commits/e2cff64674134525147896ee2f0bed6d138ad3e3

Only the reference in production code needed to be ignored. Not the one on this line.